test/help/help.test.js

```diff
@@ -154,7 +152,7 @@ describe('help', () => {
         expect(stdout).toContain('Made with ♥ by the webpack team');
     });
 
-    const commands = ['bundle', 'loader', 'plugin', 'info', 'init', 'migrate', 'serve'];
```

Comment:
Keep it

Reply:
we can but then we need to install migrate package first in test then run test on it

Reply:
will write a separate test for migrate which will preinstall the package and run it

Reply:
Yep, move this package to dev deps

Reply:
oh yes devDeps is better thanks!